packages/components/select/src/useSelect.ts

```diff
@@ -338,9 +337,7 @@ export const useSelect = (props, states: States, ctx) => {
       ) as HTMLInputElement
       const _tags = tags.value
 
-      const sizeInMap =
-        states.initialInputHeight ||
-        getComponentSize(selectSize.value || elForm.size)
+      const sizeInMap = getComponentSize(selectSize.value || elForm.size) - 2
       input.style.height =
         states.selected.length === 0
           ? `${sizeInMap}px`
```

Comment:
I think we should write `-2` here. And add a comment that it's an inner input so reduce it by 2px.